spt/features/visualizations/renderer/source files/materials_manager.cpp

```diff
@@ -4,30 +4,8 @@
 
 #ifdef SPT_MESH_RENDERING_ENABLED
 
-#include "vstdlib\IKeyValuesSystem.h"
 #include "interfaces.hpp"
 
-// We need KeyValues to make materials; they in turn need KeyValuesSystem(), but we don't link against vstdlib.
-// Here's a little hack to make that work - find the KeyValuesSystem function by searching for its symbol.
-#pragma warning(push)
-#pragma warning(disable : 4273)
-IKeyValuesSystem* KeyValuesSystem()
```

Comment:
I did mention that as part of the changes above - I need to use the KeyValues system to make materials (from the SDK), but internally that calls `KeyValuesSystem()` from within the SDK. That function only exists in vstdlib.lib, otherwise I had to patch it back in by searching for its symbol in the dll. The problem with that approach is that you get a linker warning about an externally defined symbol which I could not figure out how to suppress with pragmas.

Reply:
gotcha